nservicebus/azure-service-bus/retries.md

```diff
@@ -5,36 +5,47 @@ tags:
 - Cloud
 - Azure
 - Transport
-reviewed: 2016-09-21
+reviewed: 2016-09-22
 ---
 
-This article describes the relationship between the NServiceBus retry behavior and the Azure Service Bus native retry behavior.
-
-NServiceBus supports Immediate Retries (previously known as First Level Retries) and Delayed Retries (previously known as Second Level) at the endpoint instance level as described in the [recoverability](/nservicebus/recoverability/) article.
+This article describes the relationship between [recoverability behavior](/nservicebus/recoverability/) and the Azure Service Bus native retry behavior.
 
 Azure Service Bus supports a `MaxDeliveryCount` at the entity level, which defines how many times Azure Service Bus attempts to deliver a message before sending it to the dead letter queue. Refer to [the full configuration API](/nservicebus/azure-service-bus/configuration/full.md#controlling-entities-queues) article to learn how to adjust this setting.
 
 
-## Immediate Retries vs MaxDeliveryCount
+## Immediate Retries and MaxDeliveryCount
+
+To implement Immediate Retries, the native Azure Service Bus `MaxDeliveryCount` property of the `BrokeredMessage` is used to track the processing attempts by the endpoint instances for a specific message. If the number of attempts exceeds the `MaxRetries` setting, then the message will be forwarded to the delayed retries if configured, and eventually to at the error queue in case error persists. In case Immediate Retries are disabled, the transport will default `MaxDeliveryCount` to 10 attempts.
+
+A scaled out endpoint will retry the message by any of its instances. Once all Immediate Retries where exhausted, the message will be delegated to the Delayed Retries, if configured, or sent the error queue. So in effect, the number of immediate retries attempts will never exceed the defined number of immediate retries for an endpoint.
+
+Versions 6 and below the default value for `MaxDeliveryCount` is 6. Versions 7 and higher it has been changed to be equal to a number of immediate retries for an endpoint plus one. For system queues, such as audit and error queues, the default was changed to 10. Should endpoint disable Immediate Retries, `MaxDeliverCount` will be set to 10.
+
+
+### Keeping MaxDeliveryCount in sync with immediate retries
+
+Whenever an endpoint is redeployed with a new Immediate Retries value, endpoint specific queues, such as endpoint`s input queue, will be updated with the new value.
+
+
+For system queues, such as audit and error queues, a newly deployed or a re-deployed endpoint will not modify system queues `MaxDeliveryCount` set by end users.
```

Comment:
>For system queues, such as audit and error queues, a newly deployed or a re-deployed endpoint will not modify system queues `MaxDeliveryCount` set by end users.

Did you mean

For system queues, such as audit and error queues if a custom value was configured for `MaxDeliveryCount` redeploying the endpoint will not overwrite the already configured value.